src/test/scala/edu/ie3/util/quantities/QuantityConversionUtilsSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * © 2023. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.util.quantities
+
+import edu.ie3.simona.test.common.UnitSpec
+import edu.ie3.util.DoubleUtils.~=
+import edu.ie3.util.scala.quantities.QuantityConversionUtils.TemperatureConversionSimona
+import squants.thermal.{Celsius, Temperature}
+import tech.units.indriya.quantity.Quantities
+import tech.units.indriya.unit.Units.{CELSIUS, KELVIN}
+
+class QuantityConversionUtilsSpec extends UnitSpec {
+  implicit val doubleTolerance: Double = 1e-10
+  implicit val temperatureTolerance: Temperature = Celsius(1e-10)
+
+  "QuantityConversionUtils" should {
+    "properly convert from ComparableQuantity[Celsius] to squants temperatures and its double values" in {
```

Comment:
It is `Temperatur` instead of `Celsius`.

Reply:
True :)